cmd/goss/goss.go

```diff
@@ -33,6 +32,11 @@ func main() {
 			Usage:  "json/yaml file containing variables for template",
 			EnvVar: "GOSS_VARS",
 		},
+		cli.StringFlag{
+			Name:   "vars-inline",
+			Usage:  "json/yaml string containing variables for template (overwrites vars)",
+			EnvVar: "GOSS_VARS-INLINE",
```

Comment:
```suggestion
			EnvVar: "GOSS_VARS_INLINE",
```

Dashes aren't valid environment variables.

Reply:
Great catch! I am a bit embarrassed that I missed it. 

This reminds me that I would also like to work a bit on the integration tests. Maybe not a full rewrite of them, since multiple OSs and Architectures are fantastic. But more as an expansion of the integration testing via some kind of table test so contributors can easily add integration tests.

Reply:
Feel free to start an issue/discussion on the integration tests.. Or just create a PR with a proof of concept and we can discuss it from there.

I like the idea.

On a random/fun note.. One fun idea I've been toying with is improving the goss `command` tests and using goss to test goss.

Reply:
Exactly! I have seen something very similar done on a the @SimonBaeumer fork: https://github.com/SimonBaeumer/goss

I will see what I can put together when I have a night or 2 to play it.